src/pages/ProfilePage/ProfilePage.jsx

```diff
@@ -5,7 +5,6 @@ import ProfilePicture2 from "../../components/ProfilePicture2/ProfilePicture2";
 import ButtonSh from "../../components/shared/ButtonShort/ButtonShort";
 import TopBar from "../../components/shared/TopBar/TopBar";
 import "./ProfilePage.css";
-import BackButton from "../../components/shared/GoBackButton/GoBackButton";
```

Comment:
Is it neccessary to delete the BackButton? Are we sure we don't want to use it?

Reply:
I deleted unused imports